### ISSUE TITLE
Limited feedback from Orbit.from_horizons 

### ISSUE THREAD

**Issue by mkelley** [NASA-Planetary-Science/sbpy#204]
**High-level problem description**
I am attempting to retrieve an orbit from Horizons.  The code failed, but cannot be easily debugged.

**What did you do?**
```
from astropy.time import Time
from sbpy.data import Orbit
t = Time(['2019-01-01', '2019-02-01', '2019-03-01'])
orb = Orbit.from_horizons(399, 'majorbody', epochs=t)
```

**What did really happen?**
```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
/tmp/orbit-bug.py in <module>()
      4 
      5 t = Time(['2019-01-01', '2019-02-01', '2019-03-01'])
----> 6 orb = Orbit.from_horizons(399, 'majorbody', epochs=t)
      7 
      8 # H and G required for openorb

/home/msk/Projects/sbpy/sbpy/data/orbit.py in from_horizons(cls, targetids, id_type, epochs, center, **kwargs)
    121                            location=center, epochs=epochs)
    122 
--> 123             elem = obj.elements(**kwargs)
    124 
    125             # workaround for current version of astroquery to make

/home/msk/local/lib/python3.6/site-packages/astroquery-0.3.10.dev5546-py3.6.egg/astroquery/utils/class_or_instance.py in f(*args, **kwds)
     23         def f(*args, **kwds):
     24             if obj is not None:
---> 25                 return self.fn(obj, *args, **kwds)
     26             else:
     27                 return self.fn(cls, *args, **kwds)

/home/msk/local/lib/python3.6/site-packages/astroquery-0.3.10.dev5546-py3.6.egg/astroquery/utils/process_asyncs.py in newmethod(self, *args, **kwargs)
     27             if kwargs.get('get_query_payload') or kwargs.get('field_help'):
     28                 return response
---> 29             result = self._parse_result(response, verbose=verbose)
     30             self.table = result
     31             return result

/home/msk/local/lib/python3.6/site-packages/astroquery-0.3.10.dev5546-py3.6.egg/astroquery/jplhorizons/core.py in _parse_result(self, response, verbose)
   1282             return None
   1283         else:
-> 1284             data = self._parse_horizons(response.text)
   1285 
   1286         return data

/home/msk/local/lib/python3.6/site-packages/astroquery-0.3.10.dev5546-py3.6.egg/astroquery/jplhorizons/core.py in _parse_horizons(self, src)
   1176                                  err_msg)
   1177             else:
-> 1178                 raise ValueError(('Query failed without error message; '
   1179                                   'check URI for more information'))
   1180         # strip whitespaces from column labels

ValueError: Query failed without error message; check URI for more information
```

The message from astroquery is to check the URI, but there is no URI to check.  Orbit.from_horizons should append the URI for the user.

**Provide information on your environment:**
sbpy version: latest dev
astropy version: 3.2.1
numpy version: 1.14.5

**Comment by mkelley** [comment]
And astroquery 0.3.10.dev5546, if it matters.

**Comment by mommermi** [comment]
fixed as per #193
